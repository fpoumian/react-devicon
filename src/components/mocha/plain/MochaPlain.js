import React from 'react'
import PropTypes from 'prop-types'
import SVGDeviconInline from '../../_base/SVGDeviconInline'

import iconSVG from './MochaPlain.svg'

/** MochaPlain */
function MochaPlain({ width, height, className }) {
  return (
    <SVGDeviconInline
      className={'MochaPlain' + ' ' + className}
      iconSVG={iconSVG}
      width={width}
      height={height}
    />
  )
}

MochaPlain.propTypes = {
  className: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
}

export default MochaPlain
