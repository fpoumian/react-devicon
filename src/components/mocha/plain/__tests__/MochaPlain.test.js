import React from 'react'
import renderer from 'react-test-renderer'
import MochaPlain from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<MochaPlain />).toJSON()
  expect(tree).toMatchSnapshot()
})
