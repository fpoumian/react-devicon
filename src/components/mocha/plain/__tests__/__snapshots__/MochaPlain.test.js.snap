// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="SVGInline MochaPlain undefined"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon MochaPlain-devicon undefined-devicon\\" id=\\"Layer_1\\" data-name=\\"Layer 1\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 112 126.38\\"><defs><style>.cls-1{fill:#8d6748}</style></defs><title>mocha</title><g id=\\"plain\\"><path class=\\"cls-1\\" d=\\"M91.78 58.77l-2.3 5.42h4.61l-2.31-5.42z\\"/><path class=\\"cls-1\\" d=\\"M60.91 52h4a1 1 0 0 0 .54-.24C67.77 49 68.1 43.6 68.1 41.7a.7.7 0 0 0-.68-.7h-9a.68.68 0 0 0-.68.68c.07 1.9.34 7.36 2.65 10.08a.65.65 0 0 0 .52.24zm4.41-5.16C67 46.23 67 45.62 67 45.62a10.69 10.69 0 0 1-2 5.24.51.51 0 0 1-.41.15h-3.34c-.14 0-.34 0-.41-.15a9.6 9.6 0 0 1-1.7-3.77s3.94.56 6.18-.25z\\" transform=\\"translate(-8 -.81)\\"/><path class=\\"cls-1\\" d=\\"M116.42 29.94L67.48 1.72a6.83 6.83 0 0 0-6.83 0L11.6 29.94A6.9 6.9 0 0 0 8 35.78v56.44a7 7 0 0 0 3.58 5.83l48.94 28.22a6.83 6.83 0 0 0 6.83 0l49.05-28.21a7.14 7.14 0 0 0 3.6-5.83V35.78a7 7 0 0 0-3.58-5.84zM112 88.29a5.77 5.77 0 0 1-2.94 5l-42.2 24.37a5.65 5.65 0 0 1-5.84 0L18.87 93.31a5.63 5.63 0 0 1-2.87-5V39.58a5.71 5.71 0 0 1 2.87-5L61 10.2a5.6 5.6 0 0 1 5.82 0l42.21 24.35a5.69 5.69 0 0 1 2.93 5z\\" transform=\\"translate(-8 -.81)\\"/><circle class=\\"cls-1\\" cx=\\"40\\" cy=\\"62.53\\" r=\\"4.84\\"/><path class=\\"cls-1\\" d=\\"M108.06 36.32L65.85 12a3.87 3.87 0 0 0-1.91-.54A4.06 4.06 0 0 0 62 12L19.87 36.32A3.88 3.88 0 0 0 18 39.65v48.7a3.87 3.87 0 0 0 1.87 3.32L62 116a3.62 3.62 0 0 0 1.89.54 3.82 3.82 0 0 0 1.9-.54l42.23-24.35a3.94 3.94 0 0 0 1.98-3.3v-48.7a3.94 3.94 0 0 0-1.94-3.33zM67 68.19a4.8 4.8 0 0 0 3.65-1.69l1.52 1.5a7.7 7.7 0 0 1-1.08 1 5.9 5.9 0 0 1-1.22.75 5 5 0 0 1-1.42.47 8.33 8.33 0 0 1-1.83.14 6.72 6.72 0 0 1-4.88-2 6.1 6.1 0 0 1-1.3-2.24 7.34 7.34 0 0 1-.44-2.73v-.07a7.12 7.12 0 0 1 .42-2.71 7.63 7.63 0 0 1 1.36-2.24A6.35 6.35 0 0 1 64 56.88a7.53 7.53 0 0 1 2.83-.54 13.09 13.09 0 0 1 1.69.14c.54.14.95.27 1.42.41a6.43 6.43 0 0 1 1.15.68 11.79 11.79 0 0 1 1 .88l-1.56 1.63A4.83 4.83 0 1 0 67 68.19zm-2.83-37.63c-.34.47.07 1.63.95 3a2.83 2.83 0 0 1-.95 4 3.83 3.83 0 0 0-.75-2.31c-1.62-2.46-.13-3.61.75-4.69zm-2.71 2.51c-.27.27 0 1.09.61 2a1.91 1.91 0 0 1-.61 2.71 2.74 2.74 0 0 0-.46-1.53c-1.1-1.69-.15-2.44.46-3.18zM56.5 39h12.76a.64.64 0 0 1 .68.63c0 2.37-.34 10.37-3.74 14.14a.64.64 0 0 1-.47.23H60a.67.67 0 0 1-.47-.26c-3.37-3.86-3.7-11.74-3.7-14.08a.66.66 0 0 1 .67-.66zM21 49h34.35s.07 0 .07 1H21zm34 14.32v.07a7.37 7.37 0 0 1-.47 2.71 6.7 6.7 0 0 1-1.46 2.24 6.4 6.4 0 0 1-2.22 1.49 6.63 6.63 0 0 1-2.84.54 7.86 7.86 0 0 1-2.84-.54 6.47 6.47 0 0 1-2.24-1.49 6.3 6.3 0 0 1-1.41-2.24 7.16 7.16 0 0 1-.52-2.71v-.07a7.43 7.43 0 0 1 .53-2.71A6.79 6.79 0 0 1 43 58.37a6.46 6.46 0 0 1 2.23-1.49 6.66 6.66 0 0 1 2.85-.54 7.88 7.88 0 0 1 2.85.54 6.48 6.48 0 0 1 2.24 1.49 6.18 6.18 0 0 1 1.35 2.24 7.14 7.14 0 0 1 .48 2.71zM29.1 67H29l-4-6.46V70h-3V57h3l4.14 6.38L33.23 57H36v13h-3v-9.53zm77.9 8H21v-1h86zM86 62v-5h2v13h-2v-6h-7v6h-2V57h2v5zm6.73 8l6-13h2.17l6 13h-2.51L103 67h-6.4l-1.42 3zM107 50H70.41c0-1 .07-1 .07-1H107z\\" transform=\\"translate(-8 -.81)\\"/></g></svg>",
    }
  }
/>
`;
